macros/src/main/scala/me/rexim/morganey/meta/QuotationMacro.scala

```diff
@@ -0,0 +1,161 @@
+package me.rexim.morganey.meta
+
+import me.rexim.morganey.ast._
+
+import scala.collection.mutable.ArrayBuffer
+import scala.reflect.macros.whitebox.Context
+
+private[meta] class QuotationMacro(val c: Context) {
```

Comment:
That's a pretty cool class. :) I believe this is where all of the macro magic happens. But it has a lot of imperative logic, which I'd like to cover with any kind of tests that are run on CI. Is it possible to do that quickly? If not we can make it a separate issue and think about it later.